src/style.css

```diff
@@ -1 +1,119 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
```

Comment:
Here you are fixing the height of the container. Unless there is a specific requirement to do so, it's better to let the content size dictate the container size. This helps to avoid overflow or accidental scroll bars.